.gitignore.txt

```diff
@@ -0,0 +1,3 @@
+node_modules/
+test.md
+app_screenshot
```

Comment:
- [ ] Kindly unse a .gitignore file, in this case this file is a text file, please remove the .txt from your .gitignore so git knows wich files to ignore.